Server/shop/src/main/java/com/cashmanager/server/shop/Order.java

```diff
@@ -0,0 +1,50 @@
+package com.cashmanager.server.shop;
+
+import jakarta.persistence.*;
+import lombok.*;
+import org.hibernate.proxy.HibernateProxy;
+
+import java.util.Date;
+import java.util.Objects;
+import java.util.Set;
+import java.util.UUID;
+
+@Getter
+@Setter
+@ToString
+@RequiredArgsConstructor
+@Builder
+@AllArgsConstructor
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    @Column(name = "order_id" ,nullable = false)
+    // GenerationType instructs that a UUID for the entity should be generated automatically for us by the persistence provider.
+    private UUID orderId;
```

Comment:
Please stay simple, use only `table_id` naming for foreign keys.
Also do that for the other tables.

```suggestion
    private UUID id;
```